src/portable/st/stm32_fsdev/dcd_stm32_fsdev.c

```diff
@@ -144,6 +144,10 @@
 #  define DCD_STM32_BTABLE_LENGTH (PMA_LENGTH - DCD_STM32_BTABLE_BASE)
 #endif
 
+// Since TinyUSB doesn't use SOF for now, and this interrupt too often (1ms interval)
+// We disable SOF for now until needed later on
+#define USE_SOF     0
```

Comment:
Like the `#define`s above, I'd recommend putting a `#ifndef` around this.

Reply:
actually, we don't plan to allow application to override this value. If there is it would follow the `CFG_TUD_*` naming, so it is good as it is.

Reply:
done.